mne/preprocessing/tests/test_xdawn.py

```diff
@@ -52,7 +52,7 @@ def test_xdawn_fit():
     assert_equal(xd.correct_overlap, False)
     # no overlapp correction should give averaged evoked
     evoked = epochs['cond2'].average()
-    assert_array_equal(evoked.data, xd.evokeds_['cond2'].data)
+    assert_array_equal(evoked.data, xd.evokeds_['cond2'])
```

Comment:
Yes it is. To create `EvokedArray` data type we require `info` part. And we can't do that in `XdawnTrasnformer`. Perhaps we can make `evokeds` `EvokedArray` at the end of `fit` of `Xdawn`. What do you think?